packages/berry-fslib/sources/NodeFS.ts

```diff
@@ -226,11 +231,38 @@ export class NodeFS extends FakeFS {
     };
   }
 
-  private fromPortablePath(p: string) {
-    return p;
+  static fromPortablePath(p: string) {
+    if (process.platform !== `win32`) return p;
+
+    // Path should look like "/mnt/n/berry/scripts/plugin-pack.js"
+    // And transform to "N:\berry/scripts/plugin-pack.js"
+
+    if (!PORTABLE_PREFIX_REGEXP.test(p)) return p;
+
+    const driveLetter = p[PORTABLE_PATH_PREFIX.length].toUpperCase();
+    const pathWithoutPrefix = p.substr(PORTABLE_PATH_PREFIX.length + 1);
```

Comment:
I think at this point (where you do a regex test + manually split the string) it might be easier to just turn the regex into something like `/mnt/([a-z])(?:/(.*))?$` and use it with `match`:

```js
const [, drive, pathWithoutPrefix] = p.match(PORTABLE_PREFIX_REGEXP);
```

Reply:
Nice, I'm testing it now :wink: